python_functions.py

```diff
@@ -0,0 +1,91 @@
+# 1:
+# Create function to divide first number by the second number (numbers are parameters)
+def division(num1, num2):
+    return num1 / num2
+
+print(division(30, 4)) # 7.5
+
+print()
+
+
+# 2:
+# Create function that has list of numbers as an input and prints sum of all elements of the list
+def addition(num_list):
+    return sum(num_list)
```

Comment:
No, mohlo se to pouzit jako cviceni na for cyklus, ale nevadi :D

Reply:
Dekuji za feedback, muzu to jeste upravit :)